crowd_anki/utils/uuid.py

```diff
@@ -0,0 +1,597 @@
+r"""UUID objects (universally unique identifiers) according to RFC 4122.
```

Comment:
err, what's this about/for?

Reply:
No idea! Seems to have been a copy of [this](https://docs.python.org/3/library/uuid.html), but I don't know how it got there! It's out now.